### ISSUE TITLE
ConfigureAwait warning displayed on HTML in .cshtml file

### ISSUE THREAD

**Issue by doxxx** [renestein/ConfigureAwaitEnforcer#9]
As seen in the screenshot below, the ConfigureAwait warning is displayed on HTML in a .cshtml file. The file is part of a Blazor Server web application, in Visual Studio 2022.

![image](https://user-images.githubusercontent.com/677940/184412956-5d6c7d89-ddce-418e-a74c-a65886d9ff38.png)

**Comment by renestein** [comment]
@doxxx Thanks for the reporting.
It is weird. I think that the blazor page is "somehow" rewritten and the ConfigureAwaiteEnforcer sees 'await' call.
I will check this later (end of August/September).

**Comment by doxxx** [comment]
Somewhat related to this, there are some files (e.g. the above, razor/blazor pages, WinForms/WPF event handlers) where the default of `ConfigureAwait(true)` is correct and I don't want to add explicit `ConfigureAwait(true)` nor do I want to see the warnings. It would be nice if there was a way to ignore the warning for these files.

**Comment by renestein** [comment]
@doxxx Yes, the ConfigureAwaitEnforcer primary audience was and still is library writers (the presumption is that the analysis runs in non-UI code).
Thanks for the suggestion.

**Comment by BWcc24** [comment]
I just sumpeled over this problem, would it be possible to skip those automated created files somehow?
we mainly have libraries and backend service, but of course one project got some razor pages, and that is really annoying.

**Comment by renestein** [comment]
@BWcc24 Sorry for the inconvenience. I will try to fix this with misc.  other issues. Thanks for the patience.
